stdlib/auth/src/main/ballerina/auth/auth_provider.bal

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is this a new file? Incorrect year

Reply:
Earlier this file was `auth_store_provider.bal`. In this PR I have renamed it and updated.